EIPS/eip-4886.md

```diff
@@ -0,0 +1,122 @@
+
+    Contract / Dapp                        Register
+
+      |                                       |
+      |------------- 0x0222..---------------> |
+      |                                       |
+      | <-------nominator: 0x0222..---------- |
+      |         delivery: 0x0222..            |
+      |                                       |
+
+If the EPS register is passed the address of a Nominator it will revert. This is of vital importance. The purpose of the proxy is that the Proxy address is operating on behalf of the Nominator. The Proxy address therefore can derive the same benefits as the Nominator (for example discord roles based on the Nominator's holdings, or mint NFTs that require another NFT to be held). It is therefore imperative that the Nominator in an active proxy cannot also interact and derive these benefits, otherwise two addresses represent the same holding. A Nominator can of course delete the Proxy Record at any time and interact on it's own behalf, with the Proxy address instantly losing any benefits associated with the proxy relationship.
+
+## Rationale
+
+The rationale for this design was to provide an easy and convenient way to prove ownership of assets while putting those assets at absolutely zero risk of loss. The motivation was seeing a number of attacks making use of the fact that users need to interact with a wallet holding assets in order to derive benefits of that asset ownership.
+
+In addition to the loss of user's assets it is clear that these events damage confidence in the community in general. Users make decisions based on both actual and perceived risk, and it presents a barrier for new users accessing and using ethereum.
+
+My vision is an ethereum where users setup a new hardware wallet for assets they wish to hold long-term, then make one single contract interaction with that wallet: to nominate a hot wallet proxy. That user can always prove they own assets on that address, and they can specify it as a delivery address for new asset delivery.
+
+## Backwards Compatibility
+
+The eip is fully backwards compatible.
+
+## Test Cases
+
+The full SDLC for this proposal has been completed and it is operation at 0xfa3D2d059E9c0d348dB185B32581ded8E8243924 on mainnet, ropsten and rinkeby. The contract source code is validated and available on etherscan. The full unit test suite is available in `../assets/eip-4886/`, as is the source code and example implementations.
+
+## Reference Implementation
+
+Please see `../assets/eip-4886/contracts`
+
+
+## Security Considerations
+
+The core intention of the eip is to improve user security by better safeguarding assets and allowing greater use of cold wallet storage. 
+
+I've considered potential negative security implications and cannot envisage any. The proxy record can only become operational when a nomination has been confirmed by a proxy address, both addresses therefore having provided signed proof. 
```

Comment:
Please avoid using first and second person pronouns in an EIP.

Reply:
First and second personal pronouns removed in most recent commit. Thanks for the feedback.